src/services/applications.ts

```diff
@@ -0,0 +1,45 @@
+import firebase from "firebase/compat/app";
+import "firebase/compat/firestore";
+import "firebase/compat/auth";
```

Comment:
This looks like the old import instead of the modular version.

Reply:
Updated to move to modular imports